### PR TITLE
Minor fix on Default backend in Backend configuration.

### PR DESCRIPTION
**Pull request by francares**

### CODE REVIEW

File: src/main/scala/cromwell/backend/config/BackendConfiguration.scala [hunk 1]
```diff
@@ -27,13 +28,19 @@ object BackendConfiguration {
...
     * @return Backend configuration.
     */
-  def getDefaultBackend(): BackendConfigurationEntry = backendList.filter(_.name.equals(defaultBackend)).head
+  def getDefaultBackend(): BackendConfigurationEntry = backendList.find(_.name.equals(defaultBackend))  match {
+    case Some(backend) => backend
+    case None =>
+      val errMsg = "Default backend configuration was not found."
+      logger.error(errMsg)
+      throw new IllegalStateException(errMsg)
+  }
```
Comment (mcovarr) [inline src/main/scala/cromwell/backend/config/BackendConfiguration.scala:41]: This looks more like a `getOrElse` than a `match`

Comment (francares) [inline src/main/scala/cromwell/backend/config/BackendConfiguration.scala:41]: You're right.

**Comment by mcovarr** [comment]
:+1: after squash + rebase + thumbsup from @Horneth

**Comment by Horneth** [comment]
:+1:

**Pull request merged**